test/e2e/service_bootstrap.py

```diff
@@ -55,6 +55,8 @@ def sync_data_bucket(bucket) -> str:
                 "sync",
                 f"s3://{SAGEMAKER_SOURCE_DATA_BUCKET}",
                 f"./{temp_dir}/",
+                "--region",
```

Comment:
curious, is this a new requirement? did you find anything in S3 docs which suggests this change in behavior?

Reply:
Experienced the following error in **some but not all** regions:
![image (2)](https://user-images.githubusercontent.com/38313898/232680874-34f1b368-6dd9-453f-8e3d-2aadca06535c.png)

My speculation is there is a mismatch between region we are calling the command and the one the bucket is in. [Explicitly specify](https://docs.aws.amazon.com/cli/latest/reference/s3/sync.html) the region to use might be helpful. Some old regions might have more endpoints and can automatically switch, but for newer regions we need to explicitly specify.

References:
https://docs.aws.amazon.com/general/latest/gr/s3.html
https://repost.aws/questions/QUIZTCFPrzQXa2WJ2VaPC2lw/buckets-created-in-af-south-1-cannot-be-accessed-through-global-endpoint